doc/source/whatsnew/v2.1.0.rst

```diff
@@ -864,6 +864,7 @@ Metadata
 
 Other
 ^^^^^
+- Add complex128 to the types of numerical data we test across the test suite (:issue:`54761`)
```

Comment:
Is there an actual bug that was fixed? It would be better to describe that.

Also could you move to 2.2.0.rst if there is one

Reply:
Alas the number 54761 is the pull request itself, not an actual bug.  All this PR does it to add the complex128 datatype to the numerical data types in the test suite (and xfailing in the rare cases where there are obvious holes, such as the lack of a cython implementation for backfilling complex numbers if libs/algos.pxd).  So technically this is an ENH, except the only functionality it is adding is to the test suite.  All the complex128 machinery is already in Pandas.

Reply:
I think we decided that because there is no user-visible functionality change, that there's nothing to report in whatsnew, and I've taken that out.  So this is resolved.